src/test/java/DecodeTest.java

```diff
@@ -15,59 +21,92 @@
 /** Validate that decoding the protobuf file to json works properly. */
 @RunWith(JUnit4.class)
 public final class DecodeTest {
-  private static String inputFile;
-  private static String outputFile;
-  private static InputStream input;
-  private static OutputStream output;
+  private static InputStream encoderInput;
+  private static OutputStream encoderOutput;
+  private static String decoderOutputFile;
+  private static InputStream decoderInput;
+  private static OutputStream decoderOutput;
+  private static String[] testInputArray;
+  private static String testInput;
+  private static String decoderInputFile;
 
   @Before
-  public void setUp() {
-    inputFile = "sampleProto.sstable";
-    outputFile = "testOutput.json";
+  public void setUp() throws SecurityException, FileNotFoundException {
```

Comment:
For tests, you can just write `throws Exception`. No one will write code to call `setUp`, so there's no need to identify the specific exceptions. Same for the test cases below.